homeassistant/components/xiaomi_miio/light.py

```diff
@@ -1071,3 +1081,58 @@ async def async_update(self):
             self._brightness_pct = state_dict["brightness"]
             self._rgb = state_dict["rgb"]
             self._hs = color.color_RGB_to_hs(*self._rgb)
+
+
+class XiaomiGatewayBulb(XiaomiGatewayDevice, LightEntity):
+    """Representation of Xiaomi Gateway Bulb."""
+
+    @property
+    def brightness(self):
+        """Return the brightness of the light."""
+        return round((self._sub_device.status["brightness"] * 255) / 100)
+
+    @property
+    def color_temp(self):
+        """Return current color temperature."""
+        return self._sub_device.status["color_temp"]
+
+    @property
+    def is_on(self):
+        """Return true if light is on."""
+        return self._sub_device.status["status"] == "on"
+
+    @property
+    def min_mireds(self):
+        """Return min cct."""
+        return self._sub_device.status["cct_min"]
+
+    @property
+    def max_mireds(self):
+        """Return max cct."""
+        return self._sub_device.status["cct_max"]
+
+    @property
+    def supported_features(self):
+        """Return the supported features."""
+        return SUPPORT_BRIGHTNESS | SUPPORT_COLOR_TEMP
+
+    async def async_turn_on(self, **kwargs):
+        """Instruct the light to turn on."""
+        if not self.is_on:
```

Comment:
We normally avoid guards on state in actuator methods for polling integrations unless the device breaks when asking it to actuate the current state again. Otherwise if we have the incorrect state it's impossible to change state from Home Assistant.

Reply:
I actually do not know since I don't own such a device.
So I can't test it out.

I will remove it, and we will get some reports from users if it turns out it does not work.